common/app/views/fragments/analytics/google.scala.html

```diff
@@ -52,6 +52,30 @@
                     image.src = "@{Configuration.debug.beaconUrl}/count/pvg.gif";
                 }
             });
+
+            @*
+            If there is internal link click data in session storage, send a click event.?:
+
+            We rely on the fact that this inline JS runs before the Omniture code.
+            The Omniture code removes the entry from session storage after it has read it.
+
+            TODO: When we get rid of Omniture, we should delete the entry from session storage here.
+            *@
+
+            try {
+                var NG_STORAGE_KEY = 'gu.analytics.referrerVars';
+                var referrerVarsData = window.sessionStorage.getItem(NG_STORAGE_KEY);
+                var referrerVars = JSON.parse(referrerVarsData);
+                if (referrerVars && referrerVars.value) {
+                    var d = new Date().getTime();
+                    if (d - referrerVars.value.time < 60 * 1000) { // One minute
+                        ga('guardianTestPropertyTracker.send', 'event', 'Click', 'Internal', referrerVars.value.tag, {
```

Comment:
We can't reference `interactionTracking` from here, because this is happening in inline JS, outside of module-land. But there's no reason I need to fire the event this early. I could potentially do it in the init of `interactionTracking`.

The problem is that I need it to run before the data gets deleted from session storage [here](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/analytics/analytics.js#L202), but I can't work out when that code actually runs. `analytics.js` seems to be a bit of a weird module.